doc/team_practices.md

```diff
@@ -26,54 +74,94 @@ In the workflow below, there are multiple roles to be played:
+this workflow. That means that your PR can immediately be merged since it was
+reviewed-while-coded.
+
+**`Ready to Start`**
+
+These are stories that are ready to be worked on by developers, including
+frontend and backend work. Stories that are visually higher in a task list
+indicate a higher priority than the stories lower in the list. Although there
+are often task dependencies before moving, some stories may move directly into
+`Ready to Start` if the design/research needs have already been met.
+
+In Trello, the developer will self-assign the story by putting their face on
+the Trello card.
 
-Here is an example workflow. Unless explicitly noted, actions are assumed to be taken by the Owner.
+A developer can self-assign a story while it is in Ready to Start if they
+intend on working on that card, but developers are encouraged to avoid selecting
+stories until they are In Progress to avoid preventing others from
+attempting work on a feature.
 
-1. Click **Start** on the story in the story tracker.
+**`In Progress`**
 
-1. Create a new Git branch from master.
-If you are authorized to commit on the 18F repo, branch directly on that repo.
-Otherwise, fork the 18F repo. The branch naming convention is *storyId*-*short-description*.
-Example: `git checkout -b 123456-fix-timezones`
+When a developer starts a story, they move the card to this list and
+self-assign. Each developer should aim to have no more than 2-3 stories in
+process at a time.
 
-1. Write your code and tests and documentation. You can
-[read about running tests](https://github.com/18F/C2/blob/master/doc/setup.md#running-tests)
-and [view the doc folder](https://github.com/18F/C2/tree/master/doc).
+The author writes code and tests and documentation. Read [read about running
+tests](https://github.com/18F/C2/blob/master/doc/setup.md#running-tests) and
+[view the doc folder](https://github.com/18F/C2/tree/master/doc).
 
-1. Push your branch to Github.
+**`Code Review`**
 
-1. Create a Pull Request. The PR should reference the story URL and include a brief description
-of the changes, including any rationale for how/why the story is addressed in the way it is. Use the format
-`[#STORYID]` or `[Delivers #STORYID]` or `[Fixes #STORYID]` in the PR title.
-See the [story tracker
-documentation](https://www.pivotaltracker.com/help/api?version=v5#Tracker_Updates_in_SCM_Post_Commit_Hooks).
-Consider adding any notes relevant to testing the change, especially for live QA
+When a pull request is opened for a change, the author moves the Trello card to
+the code review list. Once a card is moved to this list, the team uses
+assignment in the GitHub pull request rather than in Trello.
+
+The PR should link to the Trello card and include a brief description of the
+changes, including any rationale for how/why the story is addressed in the way
+it is.
+
+The author should add any notes relevant to testing the change, especially for live QA
```

Comment:
should we define QA? or maybe just drop it? "live testing" makes as much sense to me as "live QA testing".